files/smail.sh

```diff
@@ -0,0 +1,62 @@
+#! /bin/bash
```

Comment:
if you can, commit the original , and different commit with your changes (if any)